Documentation/ceph-filesystem-crd.md

```diff
@@ -115,7 +115,9 @@ The pools allow all of the settings defined in the Pool CRD spec. For more detai
 
 * `metadataPool`: The settings used to create the filesystem metadata pool. Must use replication.
 * `dataPools`: The settings to create the filesystem data pools. If multiple pools are specified, Rook will add the pools to the filesystem. Assigning users or files to a pool is left as an exercise for the reader with the [CephFS documentation](http://docs.ceph.com/docs/master/cephfs/file-layouts/). The data pools can use replication or erasure coding. If erasure coding pools are specified, the cluster must be running with bluestore enabled on the OSDs.
-* `preservePoolsOnDelete`: If it is set to 'true' the pools used to support the filesystem will remain when the filesystem will be deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
+* `preserveFilesystemOnDelete`: If it is set to 'true' the filesystem will remain when the Filesystem CRD is deleted. This is a security measure to avoid accidental loss of data. It is set to 'false' by default. If not specified is also deemed as 'false'.
+
+Note: this section used to recommend the `preservePoolsOnDelete` option as an alternative way to preserve data across CRD deletion and recreation. However, this does not work; when the new filesystem adopts the preexisting metadata pool, the pool is *irrecoverably* cleared rendering *all data inaccessible*. Therefore, settting just `preservePoolsOnDelete` is no longer allowed by the operator and will result in an error during reconilation. See [this issue](https://github.com/rook/rook/issues/6492) for more details.
```

Comment:
I'd suggest no need for this paragraph, maybe just mention in the previous bullet that `preservePoolsOnDelete` should not be set. In the docs the goal is to describe current behavior, rather than giving the background for how we got there. If they want to know more background, they can always look through code or git issues.

Reply:
Done